frontend/src/metabase/components/upload/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_UPLOAD_INPUT_ID = "upload-csv";
```

Comment:
```suggestion
export const DEFAULT_UPLOAD_INPUT_ID = "upload-input";
```

this should fix a bunch of failing tests